### DISCUSSION TITLE
Where is the background picture?

### DISCUSSION THREAD

**Discussion by willyhabr** [Blackymas/NSPanel_HA_Blueprint#339] [Q&A] [answered]
Hi, please help me with one question:
How to add the cool background picture on US version?

**Comment by Blackymas** [comment, marked as answer]
Is not yet available, because I have to draw a new version for it.

The wave will come in the next updates but also for the US version, as I had already written in the release notes ;)

**Reply by willyhabr** [reply]
Thanks. You have been make a great job.
